Zend/zend_builtin_functions.c

```diff
@@ -1683,11 +1683,30 @@ static void debug_backtrace_get_args(zend_execute_data *call, zval *arg_array) /
 	if (ZEND_CALL_INFO(call) & ZEND_CALL_HAS_EXTRA_NAMED_PARAMS) {
 		zend_string *name;
 		zval *arg;
+
+		bool is_sensitive = false;
+		if (call->func->common.fn_flags & ZEND_ACC_VARIADIC) {
```

Comment:
Nit: I think that's implied by `ZEND_CALL_HAS_EXTRA_NAMED_PARAMS`: We can't have extra named params on non-variadic functions